src/test/resources/contracts/offenders.avsc

```diff
@@ -15,72 +15,52 @@
     "version": "12",
     "name": "CNOMT3"
   },
-  "propagation-policy": "1.day",
-  "retention-policy": "999.years",
+  "propagation-policy": "",
```

Comment:
We could omit these fields for now since I think we need to figure out exactly what these will contain.

Reply:
@dominic-messenger-moj mentioned to leave it blank. will remove this based on your comment now